tests/nn/test_inference.py

```diff
@@ -1473,6 +1485,155 @@ def test_sleap_track(
     args = [slp_path, "--cpu"]
     with pytest.raises(ValueError):
         sleap_track(args=args)
+        
+@pytest.mark.parametrize("tracking", ["simple", "flow", "None"])
```

Comment:
add test functions for --output argument where output can be a filename when the input is a single filename and a folder to dump the predictions in when the input is a folder